### ISSUE TITLE
Don't ask to be default mailto application on subsequent visits to /login

### ISSUE THREAD

**Issue by SethFalco** [tutao/tutanota#3947]
- [x] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [x] I've searched and did not find a similar issue.

**Bug in web app**
In the browser, Tutanota asks to be the default mailto application _every single time_ the user hits the `/login` page.

This occurs on both my laptop and phone using Firefox.

**Describe the bug**
Every time the user hits the `/login` page for Tutanota, we get a message asking:
"Add "mail.tutanota.com" as an application for mailto links?"

**To Reproduce**
1. Go to mail.tutanota.com
2. Press "x" on the message that appears
3. Login
4. Logout
5. Observe message appears again

**Expected behavior**
When a user closes that message, it shouldn't keep appearing over and over again. ^-^'

**Screenshots**
![image](https://user-images.githubusercontent.com/22801583/156933437-4b743287-a0a1-4cd3-9331-84dffb4b83e6.png)

**Desktop (please complete the following information):**
 - OS: Debian GNU/Linux 11 (bullseye) x86_64
 - Browser: Mozilla Firefox Nightly 
 - Version: 98.0a1 (2022-01-12) (64-bit)

**Smartphone (please complete the following information):**
 - Device: Pine64 PinePhone (1.2)
 - OS: postmarketOS v21.12.2 aarch64
 - Browser: Mozilla Firefox
 - Version: 91.6.0esr

**Additional context**
I haven't had this happen with other mail providers, so I'm assuming this is an issue specific to Tutanota.

**Comment by charlag** [comment]
This is how Firefox decides to implement this prompt. It will stop once you add Tutanota to the list (it will not be a default handler just because of that)

**Comment by SethFalco** [comment]
Somehow when I read it, I inserted the word "default". ^-^'
But regardless, I don't want to add the website as a handler for `mailto` either, I already have the desktop application installed.

I don't believe I've experienced this through Gmail or Outlook before.

I still think it'd be nice to add something to `localstorage` to say you've already shown users the message and not to show it again?

Otherwise, it's essentially pushing users to accept that just to get rid of the message, which isn't particularly nice.
